src/stream/src/executor_v2/mview/state.rs

```diff
@@ -80,16 +81,22 @@ impl<S: StateStore> ManagedMViewState<S> {
         let mut batch = self.keyspace.state_store().start_write_batch();
         batch.reserve(self.cache.len() * self.column_ids.len());
         let mut local = batch.prefixify(&self.keyspace);
+        let hash_builder = CRC32FastBuilder {};
 
         for (arrange_keys, cells) in self.cache.drain() {
             let row = cells.into_option();
             let arrange_key_buf = serialize_pk(&arrange_keys, &self.key_serializer)?;
             let bytes = serialize_pk_and_row(&arrange_key_buf, &row, &self.column_ids)?;
+
+            // We compute vnode on pk in materialized view since materialized views are grouped by
+            // pk.
```

Comment:
```suggestion
            // arrange keys.
```

After the concept 'arrange key' introduced by Chi, PK and arrange keys may be different